java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -1031,6 +1031,34 @@ public void zcard(BaseClient client) {
         assertTrue(executionException.getCause() instanceof RequestException);
     }
 
+    @SneakyThrows
+    @ParameterizedTest
+    @MethodSource("getClients")
+    public void zmscore(BaseClient client) {
+        String key1 = UUID.randomUUID().toString();
+        String key2 = UUID.randomUUID().toString();
+        Map<String, Double> membersScores = Map.of("one", 1.0, "two", 2.0, "three", 3.0);
+        assertEquals(3, client.zadd(key1, membersScores).get());
+        assertArrayEquals(
+                new Double[] {1.0, 2.0, 3.0},
+                client.zmscore(key1, new String[] {"one", "two", "three"}).get());
+        assertArrayEquals(
+                new Double[] {2.0, 3.0}, client.zmscore(key1, new String[] {"two", "three"}).get());
+        assertArrayEquals(
+                new Double[] {1.0, null, 3.0},
+                client.zmscore(key1, new String[] {"one", "nonExistentMember", "three"}).get());
```

Comment:
```
        assertArrayEquals(
                new Double[] {1.0, null, null, 3.0},
                client.zmscore(key1, new String[] {"one", "nonExistentMember", "nonExistentMember", "three"}).get());
```

Reply:
Don't quite get the point of the test? Every element that does not exist in the array will return a null.

Reply:
if you request the same member (that does not exist) more than once, you get more null values.

Reply:
Maybe this test makes more sense: 
```
        assertArrayEquals(
                new Double[] {1.0, 1.0, null, null},
                client.zmscore(key1, new String[] {"one", "one", "nonExistentMember", "nonExistentMember"}).get());
```